es-space-mgmt/main/default/lwc/customerList/customerList.js

```diff
@@ -11,14 +19,45 @@ export default class CustomerList extends LightningElement {
     msgForUser;
     wiredRecords;
 
-    @wire(CurrentPageReference) pageRef;
+    subscription = null;
+
+    @wire(MessageContext)
+    messageContext;
+
+    subscribeToMessageChannel() {
+        if (!this.subscription) {
```

Comment:
You don't need that check because `connectedCallback` will only be called once.

Reply:
Updated